src/interfaces/filing-interfaces/party-interface.ts

```diff
@@ -14,7 +14,7 @@ export interface ApiPersonIF {
   firstName: string
   middleName?: string
   lastName: string
-  organizationName: string
+  organizationName?: string
```

Comment:
Not sure organizationName will applicable when we are adding person as completing party

Reply:
According to the schema (ref on line 7), organizationName is only required when the partyType is "organization". And in this case, firstName and lastName are not required.

Conversely, when the partyType is "person", firstName and lastName are required, but organizationName is not.

Therefore, I'd be tempted to make firstName and lastName optional here also, perhaps with a comment that they're required for "person" (and a comment on org name too).